src/NServiceBus.Gateway.Tests/packages.config

```diff
@@ -4,6 +4,6 @@
   <package id="ApprovalTests" version="3.0.11" targetFramework="net452" />
   <package id="ApprovalUtilities" version="3.0.11" targetFramework="net452" />
   <package id="Mono.Cecil" version="0.9.6.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.0" targetFramework="net452" />
```

Comment:
should the unit tests also reference 6.1.2 (as they reference the gateway directly which is still targeting 6.0.0)?